lib/docs/src/checkbox.stories.tsx

```diff
@@ -1,27 +1,115 @@
+import { useState } from "react";
+import { DivPx, Checkbox } from "../../core/src";
+import { Utils } from "./utils";
 import { Meta } from "@storybook/react";
-import { Checkbox, DivPx } from "../../core/src";
 
-export default {
-	title: "Draft/Checkbox",
+const meta: Meta = {
+	title: "Components/Checkbox",
 	component: Checkbox,
-} as Meta;
-
-export const Primary = (): JSX.Element => (
-	<div>
-		<Checkbox indeterminate>Checkbox</Checkbox>
-		<DivPx size={8} />
-		<Checkbox defaultChecked>Checkbox</Checkbox>
-		<DivPx size={8} />
-		<Checkbox>Checkbox</Checkbox>
-		<DivPx size={8} />
-		<Checkbox disabled indeterminate>
-			Checkbox
-		</Checkbox>
-		<DivPx size={8} />
-		<Checkbox disabled defaultChecked>
-			Checkbox
-		</Checkbox>
-		<DivPx size={8} />
-		<Checkbox disabled>Checkbox</Checkbox>
-	</div>
-);
+	argTypes: {
+		checked: Utils.arg("boolean", "Visual"),
+		indeterminate: Utils.arg("boolean", "Visual"),
+		disabled: Utils.arg("boolean", "Visual"),
+		children: Utils.arg(null, "Controlled"),
+		setChecked: Utils.arg(null, "Controlled"),
+		defaultChecked: Utils.arg(null, "Uncontrolled"),
+		forwardedRef: Utils.arg(null, "Uncontrolled"),
+	},
+};
+
+Utils.page.component(meta, { sticky: true, shots: [] });
+
+export default meta;
+
+interface Props {
+	indeterminate: boolean;
+	disabled: boolean;
+	checked: boolean;
+}
+
+export const Primary = (props: Props): JSX.Element => {
+	return (
+		<>
+			<Checkbox
+				defaultChecked
+				checked={props.checked}
+				disabled={props.disabled}
+				indeterminate={props.indeterminate}
+			>
+				Checkbox
+			</Checkbox>
+			<DivPx size={8} />
+		</>
+	);
+};
+
+export const Basic = (): JSX.Element => {
+	return <Checkbox>Checkbox</Checkbox>;
+};
+
+Utils.desc(Basic)(`
+Checkbox is shown as a square box that is ticked when activated. To get start, you only need to provide a label via children:
```

Comment:
"ticked" should be changed to "checked". We are trying to be consistent :D 

> To get start, you only need to provide a label via children:

This is technically correct, but we don't want users to use it this way (uncontrolled). We should have a controlled example in the "Basic" story (one where there is a useState and use the value and setValue props)

Reply:
I'm sorry but checkbox doesn't have value and setValue props so can you be more specific about the changes that you want.